cirq-google/cirq_google/engine/simulated_local_job.py

```diff
@@ -98,22 +104,31 @@ def batched_results(self) -> List[List[cirq.Result]]:
                 raise e
         raise ValueError('Unsupported simulation type {self._type}')
 
+    def spawn_results(self):
```

Comment:
docstring? why is this called spawn?

Reply:
Added docstring and changed this to _execute_results since it should be private.